microservices/product-search/docker/docker-after-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
So if composer install will fail on something, you will never know because your container will be running right?

Reply:
Well if there is some problem with running composer, it should be discovered on CI during building branch. But you are right that in case of some unexpected error it could end that way. I expect it will be ok, but if you have some suggestion feel free to present it.